### PR TITLE
Feature -move consts outside cpp

### PR DESCRIPTION
**Pull request by Dana-1M**

### CODE REVIEW

File: RGBDPlaneDetection/plane_detection.h [hunk 1]
```diff
@@ -78,14 +64,22 @@ class PlaneDetection
...
+	
+	
+	// Camera intrinsic parameters.
+	// All BundleFusion data uses the following parameters.
+	double fx_;
+	double fy_;
+	double cx_;
+	double cy_;
+	int depthWidth_;
+	int depthHeight_;
```
Comment (yehuda-1m) [inline RGBDPlaneDetection/plane_detection.h:76]: this isnt the correct convention. Why is there a trailing `_` on the variables?

**Review by yehuda-1m — APPROVED** [review]

File: RGBDPlaneDetection/plane_detection.h (hunk 1 above)
Comment (Dana-1M) [reply RGBDPlaneDetection/plane_detection.h:76]: I tried to stick to other vars in this code, I thought this is the way of defining local class variables in cpp...

Comment (yehuda-1m) [reply RGBDPlaneDetection/plane_detection.h:76]: normally its `m_<variable> `

Comment (Dana-1M) [reply RGBDPlaneDetection/plane_detection.h:76]: To match this convention I have to change all files params, as the original code didn't use this convention....

Comment (pitz-1m) [reply RGBDPlaneDetection/plane_detection.h:76]: I've seen this convention before too... I think we can stick with what the codebase is already using, it's better than mixing different conventions in the same codebase.

**Pull request merged**